ui/pages/settings/settings.component.js

```diff
@@ -152,7 +165,7 @@ class SettingsPage extends PureComponent {
     } else if (initialBreadCrumbKey) {
       subheaderText = t(initialBreadCrumbKey);
     } else {
-      subheaderText = t(pathnameI18nKey || 'contacts');
+      subheaderText = t(pathnameI18nKey || 'general');
```

Comment:
What motivated this change?

Reply:
In the settings page container, there isn't a mapping for `SETTINGS_ROUTE`, I think whomever wrote this code was trying to default the settings route to a certain key -- "contacts" is the wrong key here. If you look in expanded view, the subheaderText shows as "Contacts" for when you first navigate to settings, it should be "General".